ELRepo patch

In newer kernels,

void generate_random_uuid(unsigned char uuid_out[16]);

is in include/linux/random.h but not in RHEL 7.6 kernels.

--- a/reiserfs.h	2015-01-08 09:58:30.000000000 -0800
+++ b/reiserfs.h	2019-01-18 15:11:25.446526391 -0800
@@ -14,6 +14,9 @@
 #include <linux/proc_fs.h>
 #include <linux/buffer_head.h>
 
+/* Added for EL 7.6 build -ay */
+void generate_random_uuid(unsigned char uuid_out[16]);
+
 /* the 32 bit compat definitions with int argument */
 #define REISERFS_IOC32_UNPACK		_IOW(0xCD, 1, int)
 #define REISERFS_IOC32_GETFLAGS		FS_IOC32_GETFLAGS
